### ISSUE TITLE
Übersetzungen 

### ISSUE THREAD

**Issue by MaikC76** [KerbalSpaceProgramDe/KSP-Deutsch-Patch#30]
Ich habe den Deutschpatch heute erstmalig installiert, daher weiss ich nicht ob manche Sachen so sein sollen oder nicht.
Auf dem bild sieht man das einiges englisch ist_
https://puu.sh/xX36f/9aeb408671.png
- Liquid fuel
- Lageregler: running
- Com first hop Dist
- Command State
- wheel authority
- Electric Charge
- Monropropellant
- Beleuchtung wechselt ständig die Sprache, wenn ich raustabbe ist es deutsch wenn ich ins Spiel tabbe englisch
Wenn ich auf ESC drücke ist im Pausemenü:
- View Key bindings
- End scenario 
nicht übersetzt
Auf der Basis:
- Die Gebäudenamen sind deutsch, alle Beschreibungen aber englisch (ausser Forschungsabteilung)

**Comment by Jeffreyveggie** [comment]
Jo.. is Normal aber das ist ja nicht so schlimm 100% wirds eh nie werden...
 

Gesendet: Freitag, 13. Oktober 2017 um 12:13 Uhr
Von: MaikC76 <notifications@github.com>
An: KerbalSpaceProgramDe/KSP-Deutsch-Patch <KSP-Deutsch-Patch@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Betreff: [KerbalSpaceProgramDe/KSP-Deutsch-Patch] Übersetzungen (#30)


Ich habe den Deutschpatch heute erstmalig installiert, daher weiss ich nicht ob manche Sachen so sein sollen oder nicht.
Auf dem bild sieht man das einiges englisch ist_
https://puu.sh/xX36f/9aeb408671.png


	Liquid fuel
	Lageregler: running
	Com first hop Dist
	Command State
	wheel authority
	Electric Charge
	Monropropellant
	Beleuchtung wechselt ständig die Sprache, wenn ich raustabbe ist es deutsch wenn ich ins Spiel tabbe englisch

**Comment by AllanScheSar** [comment]
An dieser Stelle waren wieder leider noch nicht beim debugging :(
